src/Analyzer.Core.NuGet/Analyzer.Core.nuspec

```diff
@@ -2,8 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>Azure.Templates.Analyzer.Core</id>
-    <!-- Also update version in Analyzer.Cli nuspec and Directory.Build.props. -->
```

Comment:
Were these comments removed on purpose?

Reply:
Yes, because the version here and in Analyzer.Cli.nuspec doesn't need to be updated anymore.